Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -214,6 +214,40 @@ public static class Defaults
                 ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
                 Statistic = Statistic.Maximum,
                 Namespace = AwsNamespace.Rds
+            },
+            new AlarmDefinition
+            {
+                Name = "WriteIOPSHigh",
+                Metric = "WriteIOPS",
+                Period = TimeSpan.FromMinutes(5),
+                EvaluationPeriods = 5,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
```

Comment:
You could maybe look at how dynamo alarms work. They look up the actual provisioned amount and then apply a percentage which means the alarms require less maintenance. That is one of the benefits of aws watchman, that we can have dynamic alarms.

Reply:
@ymaksym did you look at this?